styles/frame-styles.css

```diff
@@ -4,6 +4,7 @@
     --frame-background: #121212;
     --title-bar: #000000;
     --tab-text: #FFFFFF;
+    --body-background-color: #b8b8b8;
```

Comment:
So this is called `body-background-color` but I see it used as `color` instead, which sets font color. Not sure if that was your intention

Reply:
probably not.

Reply:
Fixed, but moved the PR to a new dev branch.